WikiWatershed/model-my-watershed
# Make RWD work near border of DRB

The DRB perimeter is used on the frontend to decide if a point can have RWD run on it. Previously, we used a simplified version to have a smaller payload, but this excluded some places that are inside the DRB. This PR makes it so the frontend uses a buffered (expanded by 20 miles) and simplified DRB (Delaware River Basin) perimeter to avoid this issue. It also switches to using the full DRB perimeter on the backend to maintain accuracy. 

As part of an effort to make the data wrangling process more repeatable, I've included a script and some instructions on how these new files were generated as part of the source code.

#### Testing
* Run `./scripts/debugserver.sh` and `./scripts/debugcelery.sh`.
* Try running RWD on a point that is clearly inside the DRB (eg. Philly). It should work.
* Try it on a point that is clearly outside the DRB (eg. Pittsburgh). The frontend should immediately reject it.
* Try it on Avon Grove Intermediate School, which was reported in the issue. It should work.
* Try it on a point that is inside the buffered polygon, but outside the true DRB perimeter (eg. Lancaster). It should start RWD, but then server should respond with an error.

Connects #1583
